### PR TITLE
[Cody Web] add feedback collection

### PR DESCRIPTION
**Pull request by thenamankumar**
Add VS Code like feedback button to Cody Chat on web.

image

## Test plan

- Visit /cody

### CODE REVIEW

**Comment by sourcegraph-buildkite** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.01% (+1.49 kb) | 0.01% (+1.49 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/naman/cody-web-feedback/compare-report.html;tab=live_object) for a full comparison between the commits 23f9a2c and f520c48 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

**Review by vdavid — APPROVED** [review]
Nice and elegant, looks good to me

File: client/web/src/cody/components/ChatUI/ChatUi.module.scss [hunk 1]
```diff
@@ -122,3 +130,18 @@
 :global(.cody-chat-error) > span {
     font-weight: bold;
 }
+
+.feedback-buttons-wrapper {
+    button {
+        margin-left: 0.25rem;
+        padding: 0.25rem;
+
+        svg {
```
Comment (eseliger) [inline client/web/src/cody/components/ChatUI/ChatUi.module.scss:139]: could this be tied less to the HTML markup by using classnames instead?

File: client/web/src/cody/components/ChatUI/ChatUi.tsx [hunk 2]
```diff
@@ -82,6 +93,37 @@ export const EditButton: React.FunctionComponent<EditButtonProps> = ({
...
+            setFeedbackSubmitted(true)
+        },
+        [feedbackButtonsOnSubmit]
+    )
+
+    return (
+        <div className={classNames('d-flex', styles.feedbackButtonsWrapper)}>
+            {feedbackSubmitted ? (
+                <Button title="Feedback submitted." disabled={true}>
+                    <Icon aria-label="Scroll down" svgPath={mdiCheck} />
```
Comment (eseliger) [inline client/web/src/cody/components/ChatUI/ChatUi.tsx:111]: scroll down? what does that mean in this context?

**Review by eseliger — APPROVED** [review]

File: client/web/src/cody/components/ChatUI/ChatUi.module.scss (hunk 1 above)
Comment (thenamankumar) [reply client/web/src/cody/components/ChatUI/ChatUi.module.scss:139]: done

File: client/web/src/cody/components/ChatUI/ChatUi.tsx (hunk 2 above)
Comment (thenamankumar) [reply client/web/src/cody/components/ChatUI/ChatUi.tsx:111]: Silly copy paste.

**Pull request merged**